spec/mailers/candidate_mailer/candidate_mailer_application_rejected_spec.rb

```diff
@@ -394,4 +394,30 @@
       expect(email.body).to have_no_content 'There are still courses with placements available'
     end
   end
+
+  describe 'tailored teacher training adviser text for "assigned" adviser status' do
+    let(:application_choice) { create(:application_choice, :rejected, rejection_reason: 'Missing your English GCSE', course_option:) }
+    let(:email) { described_class.application_rejected(application_choice) }
```

Comment:
```suggestion
    subject(:email) { described_class.application_rejected(application_choice) }
```
Does this change still work? I think it makes it read a little clearer, what do you think?

Reply:
Agree, and yes still passes!